### PR TITLE
MAINT: add workaround for doctesting with new numpy repr

### PR DESCRIPTION
**Pull request by bsipocz**
This should close https://github.com/astropy/pyvo/issues/464 and fix the CI failure we see in the devtest job.

### CODE REVIEW

File: pyvo/conftest.py [hunk 1]
```diff
@@ -15,6 +16,8 @@
 except ImportError:
     ASTROPY_HEADER = False
 
+# Keep this until we require numpy to be >=2.0
+np.set_printoptions(legacy="1.25")
```
Comment (pllim) [inline pyvo/conftest.py:20]: Do you need a numpy version check before calling it?

Comment (bsipocz) [reply pyvo/conftest.py:20]: CI says that apparently I do 😅

File: pyvo/conftest.py [hunk 2]
```diff
@@ -7,14 +7,19 @@
 
 """
 
+import numpy as np
```
Comment (pllim) [inline pyvo/conftest.py:10]: I think this numpy import can go inside the "if" ?

Comment (pllim) [reply pyvo/conftest.py:10]: Ah, nvm... I see that you using it for version compare, ignore me.

Comment (bsipocz) [reply pyvo/conftest.py:10]: It doesn't matter anyway as we import numpy as soon as we touch astropy.

**Comment by bsipocz** [comment]
Note that online test failure is expected and is taken care of in a separate PR already.

**Comment by bsipocz** [comment]
(messing with the changelog label to test the GHA)

**Pull request merged**